docs/README.md

```diff
@@ -228,6 +229,21 @@ The valid values for this option are:
 * If you configure the LogDNA Agent to run as non-root, review the [documentation](KUBERNETES.md#enabling-file-offset-tracking-across-restarts) about enabling "statefulness" for the LogDNA Agent.
 * When upgrading from LogDNA Agent version 3.0 to 3.1, the state file will initially be empty, so the lookback setting will be used for existing files. After that (i.e. on process restart), the state file will be present and will be used.
 
+### Configuring Lease Startup
+
+The lease startup configuration uses Kubernetes Leases to limit the number of agents that can start at one time on a cluster. When enabled, the agent will "claim" a lease before starting. Once started, the agent will then release the lease. If no leases are available, the agent will wait for one to become available. This feature would only be needed if running the agent on a cluster large enough that you'd risk crashing `etcd` if all the the agents tried to connect at once.
```

Comment:
Should we give a number of what large means? We know 500 is a large number.

Reply:
I left a bit vague just because I don't think we know the exact number at which issue will occur and it would probably depend on a number of factors.

Perhaps @alfeng6 has some thoughts on this?

Reply:
As Agent  consumes some resources here would be useful to stress-test this. Like on empty cluster with N nodes to know the limits and base line.

Reply:
As per discussion, we're going to perform additional scale testing after merge.